workflow/schemas/config.schema.yaml

```diff
@@ -18,3 +18,51 @@ properties:
       required:
         - name
         - metadata
+      properties:
+        name:
+          type: string
+          minLength: 1
+        metadata:
+          type: string
+          minLength: 1
+        sequences:
+          type: string
+          minLength: 1
+        aligned:
+          type: string
+          minLength: 1
+        to-exclude:
+          type: string
+          minLength: 1
+        masked:
+          type: string
+          minLength: 1
+        filtered:
+          type: string
+          minLength: 1
+      additionalProperties: false
+
+  builds:
+    type: object
+    minProperties: 1
+    propertyNames:
+      # Allow build names to contain alpha characters, underscores, and hyphens
+      # but not special strings used for Nextstrain builds.  Also used in the
+      # workflow's wildcard_constraints.
+      pattern: "^(?:[_a-zA-Z-](?!(tip-frequencies)))+$"
+
+  S3_DST_COMPRESSION:
+    type: string
+    enum:
+      - gz
+      - xz
+
+  S3_DST_ORIGINS:
+    type: array
+    minItems: 1
+    items:
+      type: string
+      # Note: The {origin} wildcard in workflow rules includes a leading
```

Comment:
PR #615 removed this so that `{origin}` is a "normal" wildcard and the `_` character is specified in the input/output parameters of each rule.

Reply:
Ah! Rebased onto latest `master` and updated the comment.